datalad/support/annexrepo.py

```diff
@@ -2429,7 +2429,13 @@ def info(self, files, batch=False, fast=False):
         # and that they all have 'file' equal to the passed one
         out = {}
         for j, f in zip(json_objects, files):
-            assert(j.pop('file') == f)
+            # starting with version of annex around
+            # 8.20200330+git119-gcee6b344b
```

Comment:
specifically 957a87b43 (fix absolute filenames fed into --batch and git-annex info, 2020-04-15)

Reply:
adjusted the comment to point specifically